services/scheduled_message_service.py

```
@@ -0,0 +1,143 @@
+"""
+ScheduledMessageService: Encapsulate scheduled message CRUD with Pydantic params.
+
+Consolidates the loose scheduled-message functions from api.py into a single
+service with typed parameter models and clear method names.
+"""
+
+from datetime import datetime
+from collections.abc import AsyncIterator
```

Flake8 (7.3.0)

[error] 'collections.abc.AsyncIterator' imported but unused

(F401)